src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { Post } from '../../types/Post';
```

Comment:
Consider importing `Post` type from a separate file to avoid duplication and improve maintainability. 🤝